### PR TITLE
asn/add clear alarm

### PR DESCRIPTION
**Pull request by artnaseef**
## JIRA Link
https://issues.opennms.org/browse/HS-xx

## Checklist
* [ ] JIRA has been updated to 'in review'.
* [ ] Unit tests have been added as part of the PR.
* [ ] Manual tests have been completed, and notes added to JIRA.
* [ ] Details have been provided to docs team (if necessary)
* [ ] Screenshot(s) included (applicable to changes with a visual impact).

### CODE REVIEW

**Comment by yangli2014** [comment]
@artnaseef When I test the add ack endpoint I got the following error
`
[horizon-stream-core] java.lang.IllegalStateException: No acknowlegables in the database for ack: Acknowledgment ID:null User:admin Time:Fri May 13 13:55:47 GMT 2022 AckType:Alarm AckAction:Acknowledge Acknowledable ID:3
[horizon-stream-core] 	at org.opennms.horizon.db.dao.impl.AcknowledgmentDaoHibernate.processAck(AcknowledgmentDaoHibernate.java:103)
`
After I check the database, we need to store an Ack entity first. However, the acks table is empty.

File: platform/alarms/rest/src/main/java/org/opennms/netmgt/alarmd/rest/internal/AlarmRestServiceImpl.java [hunk 1]
```diff
@@ -200,6 +215,38 @@ public Response getAlarms(@Context final SecurityContext securityContext, @Conte
...
+
+            updateAlarmTicket(id, alarmAck);
+
+            return "acknowledged";
+        });
+    }
+
+    @DELETE
+    @Path("{id}/ack")
+    @Produces(MediaType.TEXT_PLAIN)
```
Comment (yangli2014) [inline platform/alarms/rest/src/main/java/org/opennms/netmgt/alarmd/rest/internal/AlarmRestServiceImpl.java:237]: Same as above

File: platform/alarms/rest/src/main/java/org/opennms/netmgt/alarmd/rest/internal/AlarmRestServiceImpl.java [hunk 2]
```diff
@@ -200,6 +215,38 @@ public Response getAlarms(@Context final SecurityContext securityContext, @Conte
 
     }
 
+    @POST
+    @Path("{id}/ack")
+    @Produces(MediaType.TEXT_PLAIN)
```
[the remark this answers is not in the document]
Comment (yangli2014) [reply platform/alarms/rest/src/main/java/org/opennms/netmgt/alarmd/rest/internal/AlarmRestServiceImpl.java:220]: I modified it with 
`@Produces({MediaType.TEXT_PLAIN, MediaType.APPLICATION_JSON})`

**Comment by yangli2014** [comment]
@artnaseef you can merge it now.

**Comment by artnaseef** [comment]
This work was included in another PR.

**Pull request closed without merging**